### PR TITLE
Updating chapter 15

### PR DESCRIPTION
**Pull request by andreaswittig**

### CODE REVIEW

File: chapter15/url2png/config.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "QueueUrl": "https://queue.amazonaws.com/123456789/url2png",
-  "Bucket": "url2png-$yourname"
+  "QueueUrl": "https://sqs.us-east-1.amazonaws.com/486555357186/url2png",
```
Comment (michaelwittig) [inline chapter15/url2png/config.json:2]: is this file changed by accident?

**Review by michaelwittig — CHANGES_REQUESTED** [review]

File: chapter15/url2png/index.js [hunk 2]
```diff
@@ -23,6 +21,6 @@ sqs.sendMessage({
   if (err) {
     console.log('error', err);
   } else {
-    console.log('PNG will be soon available at http://' + config.Bucket + '.s3-website-us-east-1.amazonaws.com/' + id + '.png');
+    console.log('PNG will be soon available at http://' + config.Bucket + '.s3.amazonaws.com/' + id + '.png');
```
[the remark this answers is not in the document]
Comment (michaelwittig) [reply chapter15/url2png/index.js:24]: ok. scheint auch so zu funktionieren

File: chapter15/url2png/config.json (hunk 1 above)
Comment (andreaswittig) [reply chapter15/url2png/config.json:2]: Ups. 🙈

File: chapter15/url2png/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (andreaswittig) [reply chapter15/url2png/index.js:24]: Might work for `us-east-1` only. Will fix.

[the remark this answers is not in the document]
Comment (andreaswittig) [reply chapter15/url2png/index.js:24]: Tested for `eu-west-1`. Works fine.

**Pull request merged**